processor/tailsamplingprocessor/processor.go

```diff
@@ -284,8 +290,8 @@ type policyDecisionMetrics struct {
 }
 
 type policyMetrics struct {
-	idNotFoundOnMapCount, evaluateErrorCount, decisionSampled, decisionNotSampled, decisionDropped int64
-	tracesSampledByPolicyDecision                                                                  []map[sampling.Decision]policyDecisionMetrics
+	idNotFoundOnMapCount, evaluateErrorCount, decisionSampled, decisionNotSampled, decisionDropped, decisionSkipped, decisionContinued int64
```

Comment:
This should most likely be moved to seperate line at this point lol. 

There isn't a linting rule for this, so please treat it as a nit and not something need to fix in this PR.

Reply:
fixed